apps/omg_eth/lib/omg_eth/transact.ex

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Eth.Transact do
```

Comment:
not the best name but can't figure out how to name it properly :( cc @unnawut

Reply:
Sounds clear enough to me. Maybe just change to the noun form `Transaction`?